### DISCUSSION TITLE
vendor/autoload.php

### DISCUSSION THREAD

**Discussion by lambright** [vichan-devel/vichan#603] [General]
I'm so sorry to ask this but I'm lost about this auto load (vendor/autoload.php) it's not in the download and I don't know how to get this thing working. Thanks for any insight I did search and couldn't find anything.

**Comment by perdedora** [comment]
You need to follow these instructions: https://github.com/vichan-devel/vichan#installation

**Reply by lambright** [reply]
It says to.... run composer install inside the directory.   
I am ssh into my server, I typed after my [server]$. composer install 
<img width="513" alt="Screen Shot 2023-07-20 at 10 15 10 PM" src="https://github.com/vichan-devel/vichan/assets/435028/5b9f7378-93c5-4d78-a4a2-03125b178547">

**Reply by perdedora** [reply]
first you need to install composer https://getcomposer.org/download/ then run `composer install`

**Comment by lambright** [comment]
Can somebody help me please.

**Comment by lambright** [comment]
What I get
<img width="1094" alt="Screen Shot 2023-07-20 at 10 29 46 PM" src="https://github.com/vichan-devel/vichan/assets/435028/26f3a5bb-162a-485e-9ae9-205ee5d4e22b">

**Comment by lambright** [comment]
I needed to do php composer.json install
